### ISSUE TITLE
V1 Badges List

### ISSUE THREAD

**Issue by amiller-gh** [nodejs/badges#8]
We will need to determine what content will constitute our 1.0 launch of Node.js Badges. These badges should follow the process decided upon in https://github.com/nodejs/badges/issues/4, but for the V1 release of badges, having a list of 5-10 inaugural badges to iterate on will help with this initial launch and iterating on the style guide (see: https://github.com/nodejs/badges/issues/7).

What would you like to see in a first set of badges? Ideas already thrown out elsewhere include:
 - Node.js Team / [Working Group](https://github.com/nodejs/build/issues/529) membership badges
 - Event specific badges
 - Contributor milestone badges (ex: first contribution, "power" contributor, etc...)

**Comment by Tiriel** [comment]
I fear the initial badge list will be a bit more than 5-10 badges :sweat_smile: 

I think the basis for V1 should definitely include First contrib and WG/Initiatives badges.

And for the followup, I'll ask again because that's a subject I really want to work on, but I think we should get "diversity" versions of the badges. At least something like special "pride editions" of the usual badges.

**Comment by bnb** [comment]
I'd be concerned that going over 5-10 badges may not be beneficial for the sustainability of this initiative. Committing to more than that (even committing to 10, to be honest) is going to be a non-trivial amount of work, and with the limited number of contributors we should make sure we're scoping to the amount of work that's possible with the contributors we have.

**Comment by amiller-gh** [comment]
Agreed with @bnb on keeping scope-of-work reasonable – but it also doesn't hurt to dream big! Lets make sure to stack rank all badge ideas. We can prioritize the top 5-10 for v1 and leave the rest in the queue for implementation later. That active list should be captured somewhere in this repo.

@Tiriel, love the diversity badges idea. We'll need a classification for badges that can be claimed with no strings attached. (Side note: Is Women in Node a thing yet? The acronym is too good to pass up – WIN!)

My pitch for first five:
 - First Time Contributor
 - Collaborator
 - Technical Steering Committee Member
 - Community Committee Member
 - [Node Summit 2018 Attendee](http://www.nodesummit.com/)? Maybe? Would be great for visibility.

Once we get movement on https://github.com/nodejs/badges/issues/4 I'll be sure to get all ideas discussed in this issue PR'd as an RFC. Until then, keep them coming!

**Comment by Tiriel** [comment]
Let's just be clear, I was not saying we should release 50+ badges from the start. I was just pointing we should be prepared to have to make choices on which to work on first, because de list will be larger than 5-10.

I really like the WIN idea, but we should first get the opinion of the ones concerned I think. For the first list, the first 4 seem great to me, I'm not sure about the last one. But hey, why not.

**Comment by maddhruv** [comment]
I agree with @Tiriel 
there could be more than 10 badges even in the V1 if we are not very limited with the scale of V1 badges.
more of the badges that can be in V1 - 
- i18n translation
- voted in a _x_ release of node
- voted in committee elections (board)
- attended NodeSummit
- publish packages (maybe)
- contributed to nodejs documentations
- node contributor (_made 50 commits to nodejs/node_)
- nodejs member for over 1 year
- member of nodejs board/CommComm
- member of security team
- designed 3 badges :stuck_out_tongue_winking_eye: 

There could be more/less of them, this is just an idea how vast we could go or stop with the badges selection

**Comment by mhdawson** [comment]
I'd also see one for each of the working groups.

**Comment by bnb** [comment]
@mhdawson I'd definitely love to see one for each of the WGs as well.

@amiller-gh I suppose this comes down to how we want to approach the first batch - do we want to ship a few based on a framework and let others jump in with theirs, or do we want to officially push all of them out in batches?

**Comment by amiller-gh** [comment]
The 5-10 number was really just an off-the-cuff guesstimate of what we'd need to properly develop a design framework that can support a more incremental development process. In my opinion priority #1 should be to develop the process and set ourselves up for rolling badge releases. That way we can let the velocity of this team / the community dictate the pace!

My intent was rolling releases – I'm only a fan of big-bang releases if we have the guaranteed resourcing to do it well 🙂

It may help us to have a deadline for having the framework developed and tested. I like the idea of using the Node Summit in Berlin to deliver/hype badges v1 – should we target developing committee/WG designs first so we can engage people already in the community at the summit?

Here is the current list of [active TSC working groups](https://github.com/nodejs/TSC/blob/master/WORKING_GROUPS.md) according to the WG readme:

 - Website
 - Streams
 - Build
 - Diagnostics
 - i18n
 - Docker
 - Addon API
 - Benchmarking
 - Post-mortem
 - Release

@mhdawson, are there any groups missing here that you can think of?

**Comment by ryzokuken** [comment]
Plus again, as I also mentioned in the meeting, I'd want us to prioritize working on badges for current and preferably newer contributors. `node emeriti` does not serve a huge purpose IMHO, considering that badges would be much more coveted among newer contributors than veterans.

**Comment by maddhruv** [comment]
@bnb yeah sure!
And also for some working groups? Like which all to include!

**Comment by maddhruv** [comment]
I will create a PR by tomorrow about the final idea for v1 badges list, and keep the suggestions and proposals down there

**Comment by softwarecoder** [comment]
We should also have Badge category like Bronze, Sliver, Gold & Platinum.
